layouts/partials/fragments/contact.html

```diff
@@ -33,11 +33,18 @@ <h2>{{- . | markdownify -}}</h2>
     </div>
     <div class="row">
       <div class="col-12">
-        {{ if not $contact.posturl }}
-          <form class="contact" action="https://formspree.io/{{ $contact.email }}" method="POST" name="sentMessage" id="{{ $contact.formName | default "contactForm" }}">
-        {{ else }}
-          <form class="contact" action="{{ $contact.postURL }}" method="POST" name="sentMessage" id="{{ $contact.formName | default "contactForm" }}">
-        {{ end }}
+        <form class="contact" action="
+          {{- if not $contact.netlify -}}
+            {{- $postUrl := $contact.posturl | default $contact.email -}}
+            {{- printf "https://formspree.io/%s" $postUrl -}}
```

Comment:
This would always use formspree afaik. Even with `postUrl = "mailout.example.com"` it would use `https://formspree.io/mailout.example.com`